packages/core/src/process/4-MutationTestExecutor.ts

```diff
@@ -76,6 +77,14 @@ export class MutationTestExecutor {
     };
   }
 
+  private executeIgnore(input$: Observable<MutantTestCoverage>) {
+    const [notIgnoredMutant$, ignoredMutant$] = partition(input$.pipe(shareReplay()), ({ mutant }) => mutant.ignoreReason === undefined);
+    return {
+      ignoredResult$: ignoredMutant$.pipe(map(({ mutant }) => this.mutationTestReportHelper.reportMutantIgnored(mutant))),
```

Comment:
Could you format this to multiple lines? Right now, it's not immediatly visible that the `notIgnoredMutant$` is outside the pipe operator.

For example:
```ts
    return {
      ignoredResult$: ignoredMutant$.pipe(
        map(({ mutant }) => this.mutationTestReportHelper.reportMutantIgnored(mutant))
      ),
      notIgnoredMutant$,
    };
```

Reply:
I do agree with this ^ looks much cleaner

Reply:
Thanks, will do that before merging. Will do that for all observables in this file